.github/workflows/all.yml

```
@@ -171,6 +171,11 @@ jobs:
               && go install)
           sudo apt-get update -y
           sudo apt-get install -y fakeroot rpm
+      - name: Generate Windows binary metadata
+        run: |
+          (cd && go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo)
```

```suggestion
          (cd && go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo@1.2.0)
```
we probably should pin the version we install, just in case...

---

Resolved in latest push.